samples/basic-component/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "sample-basic-component",
+    "version": "0.0.0",
+    "private": true,
+    "scripts": {
+        "build": "vertigis-web-sdk build",
+        "start": "vertigis-web-sdk start"
+    },
+    "dependencies": {
+        "@geocortex/web": "^5.6.0"
```

Comment:
These samples are guaranteed to break the moment we roll out 5.7.0. It might be best to lock the version for now until we get around to updating them after 5.7 is released.

Reply:
Then again they'll break anyway since we don't have versions in SaaS. But in this case they wouldn't even compile anymore without errors. Maybe it's a non-issue if you have a PR ready to go?

Reply:
Unfortunately they'll break either way because the runtime will explode :frowning_face:

Reply:
Sorry when i replied previously your second message wasn't showing. Yes agreed i think we can queue up some PRs closer to 5.7 release for Dev Center, SDK, and samples.